### PR TITLE
Adds suggested features.

### LINKED ISSUE #43: Bug: Multiple issues
1. Navbar is broken on some pages
2. URL helper function (the one that adds a protocol in its absence) is not working
3. HTML is broken on a lot of pages
4. `Remarks to user` is an editable text field
5.  Cancel button appears above the table in user's `my complaints`
6. `My complaints` does not show the `url unblock requests`.

### PR DESCRIPTION
**Pull request by 0xVikas**
Added image upload option
Added Date field
Removed cancel button in Handle_complaint.html
Removed Wifi Option in category.
Close #43

### CODE REVIEW

File: complaints/templates/complaints/previous_requests.html [hunk 1]
```diff
@@ -128,41 +117,58 @@
...
                   <tr style="display:none;" id="{{ req.pk }}ra">
                      <td style="background:#EEEEEE;">Details:</td>
-                     <td style="background:#EEEEEE;" colspan="2"><textarea class="form-control" a="true" disabled
-                           rows="5" cols="2">{{ req.reason }}</textarea></td>
-                     <td rowspan="2" style="background:#EEEEEE;">
+                     <td style="background:#EEEEEE;" colspan="2"><textarea class="form-control" a="true" disabled rows="5" cols="2">{{ req.reason }}</textarea></td>
+                     <td rowspan="3" style="background:#EEEEEE;">
                         {% if user.is_staff %}
-                        <form method="post" action="{% url 'cancel-unblock-request' %}">
+                        <form method="post" action="{% url 'update_request_resolve' %}">
```
Comment (Aviral14) [inline complaints/templates/complaints/previous_requests.html]: This if block is not required.

File: complaints/templates/complaints/previous_requests.html [hunk 2]
```diff
@@ -1,92 +1,81 @@
...
+                     </tr>
+                     <div>
+                        <tr style="display:none;" id="{{ comp.pk }}a">
+                         <td style="background:white;">Details:</td>
+                         <td style="background:white;" colspan="2"><textarea class="form-control" a="true" name="remark" disabled rows="5" cols="2">{{ comp.remark }}</textarea></td>
+                        
+                         <td style="background:white;" colspan="1" rowspan="5">Staff's Remarks:<textarea class="form-control" a="true" id="remarktouser" name="remark_to_user" {% if user.is_staff == False and user.is_nucleus == False %} disabled {%endif%} rows="5" cols="2" placeholder="Staff's remarks">{{ comp.remark_to_user }}</textarea></td>
+                         <td rowspan="5" style="background:white;">
+                         {% if comp.status == comp.REGISTERED %}
+                         <form method="post" action="{% url 'complaint-resolve' %}">
```
Comment (Aviral14) [inline complaints/templates/complaints/previous_requests.html]: The url is incorrect. Use 'cancel-complaint'

**Comment by Kushagra-0801** [comment]
There are way too many commits here. Do an interactive rebase and make four commits total, one for each task

File: complaints/static/main.js [hunk 3]
```diff
@@ -13,6 +13,23 @@ function urg_fun() {
 		}
 	}
 	
+function date_fun(){
```
Comment (Kushagra-0801) [inline complaints/static/main.js]: What the fuck is happening here?
You are setting the minimum value for date to be 1 month in the future and maximum value can have day 0 and cannot have day 28?

Comment (0xVikas) [reply complaints/static/main.js]: .. just forget about that I didnt use that function at all

File: complaints/templates/complaints/previous_requests.html (hunk 2 above)
Comment (Aviral14) [reply complaints/templates/complaints/previous_requests.html]: Display the image uploaded by the user!!

File: complaints/static/main.js (hunk 3 above)
Comment (Kushagra-0801) [reply complaints/static/main.js]: then fucking remove it

File: complaints/templates/complaints/handle_complaints.html [hunk 4]
```diff
@@ -38,19 +39,15 @@
                <tbody bgcolor="white">
                   <tr>
                      <td>{{ comp.category }}</td>
-                     <!--<td>{{ comp.contact_no }}</td>-->
-                     <!--<td>{{ comp.remark }}</td>-->
-                     <!--<td>{{ comp.uploaded_at }}</td>-->
+
```
Comment (Aviral14) [inline complaints/templates/complaints/handle_complaints.html]: Display the image to the handler also!

File: complaints/templates/complaints/handle_complaints.html [hunk 5]
```diff
@@ -142,7 +124,8 @@
                {% endfor %}
             </table>
          </center>
-         <p><b>Status</b> : RD: Registered; CD: Cancelled;</p>
+         <p><b>Status</b> : RD: Registered; CD: Cancelled; DN: Done/Resolved</p>
```
Comment (Kushagra-0801) [inline complaints/templates/complaints/handle_complaints.html]: Where's the taken up status?

File: complaints/templates/complaints/previous_requests.html [hunk 6]
```diff
@@ -1,92 +1,75 @@
...
-   <li><a href="{% url 'account-logout' %}"><i class="fa fa-power-off"></i> Logout</a></li>
-   {%else%}
-   <li><a href="{% url 'account-login' %}"><span class="glyphicon glyphicon-user"></span> Login</a></li>
-   {%endif%}
-</ul>
-</div>
-</div>
+	{% if user.is_authenticated %} 
+	<li><a href="{% url 'previous-requests' %}">My Complaints</a></li>
+        <li class=""><a href="{% url 'complaint-register' %}">Register Complaints</a></li>
```
Comment (Kushagra-0801) [inline complaints/templates/complaints/previous_requests.html]: whats this here for?

File: complaints/templates/complaints/previous_requests.html [hunk 7]
```diff
@@ -128,41 +111,48 @@
...
                   <tr style="display:none;" id="{{ req.pk }}ra">
                      <td style="background:#EEEEEE;">Details:</td>
-                     <td style="background:#EEEEEE;" colspan="2"><textarea class="form-control" a="true" disabled
-                           rows="5" cols="2">{{ req.reason }}</textarea></td>
-                     <td rowspan="2" style="background:#EEEEEE;">
-                        {% if user.is_staff %}
+                     <td style="background:#EEEEEE;" colspan="2"><textarea class="form-control" a="true" disabled rows="5" cols="2">{{ req.reason }}</textarea></td>
+                     <td rowspan="3" style="background:#EEEEEE;">
+                        Staff's Remarks:<textarea class="form-control" {% if user.is_staff == False %} disabled {%endif%} a="true" name="remark_to_user" rows="5" placeholder="Staff's remarks">{{req.remark_to_user}}</textarea>
+			{% if user.is_nucleus %}
```
Comment (Aviral14) [inline complaints/templates/complaints/previous_requests.html]: This block has no use here

File: complaints/templates/complaints/handle_complaints.html (hunk 5 above)
Comment (0xVikas) [reply complaints/templates/complaints/handle_complaints.html]: What's the status for taken up?

**Comment by Kushagra-0801** [comment]
The code seems good but there are too many commits. Do an interactive rebase and reduce the commits to no more than 6 and no less and 3. Use git add --patch for committing partial files. Ask @Aviral14 if you have any questions

**Pull request merged**